src/index.css

```diff
@@ -1,7 +1,276 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --pale-gray: #f5f6fa;
+  --white-dark: #fafafa;
+  --white-light: #fff;
+  --body-back: rgb(237, 234, 234);
+  --blue-azure: #0290ff;
+  --gray-dark: #888;
+  --black-dark: #121212;
+}
+
+h1 {
+  font-size: x-large;
+}
+
 body {
   font-family: 'Montserrat', sans-serif;
+  background-color: var(--pale-gray);
+  height: 100%;
+}
+
+.container {
+  width: 90%;
+  margin: auto;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.3);
+}
+
+header,
+.list .books li {
+  background-color: var(--white-light);
+}
+
+header {
+  margin: 5%  auto 0 auto;
+  width: 100%;
+  padding: 20px 0;
+}
+
+nav {
+  width: 85%;
+  margin: auto;
+  justify-content: space-between;
+}
+
+nav h1 {
+  color: var(--blue-azure);
+}
+
+nav,
+nav ul {
+  align-items: center;
+}
+
+nav,
+nav ul,
+.list .books .book-card,
+.list .books li .book-card .button-card,
+.form-add-book,
+.card-layout,
+.percentage {
+  display: flex;
+}
+
+nav ul {
+  gap: 20px;
+}
+
+nav ul li a {
+  color: var(--gray-dark);
+}
+
+nav ul li .active {
+  color: var(--black-dark);
+  font-weight: 500;
+}
+
+nav button {
+  border-radius: 50%;
+  padding: 10px 12px;
+  border: none;
+}
```

Comment:
- [ ] Kindly move the items labelled 1 and 2 to the position shown in the image. Also please ensure that the background of the part labelled 3 has the same background colour as in the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
- [ ] Also, kindly change `Home` to `Books` as in the given design and make the two nav items uppercase

### Zeplin design
![Screenshot 2023-06-21 at 16 08 38 (2)](https://github.com/zbahati/BookStore-React/assets/64774936/5e430224-e232-4429-9faf-221be5647958)

### Your design
![Screenshot 2023-06-21 at 15 42 05 (2)](https://github.com/zbahati/BookStore-React/assets/64774936/822f2ad7-90ca-43ee-a67c-d4ea08520718)